folder-management-plugin/grails-app/views/fmFolder/_selectVendors.gsp

```diff
@@ -1,4 +1,8 @@
 
 <g:select style="width: 400px" id="vendor" name="vendor" noSelection="${['null':'Select...']}" from="${vendors}" value="${vendor}"
-			onchange="${remoteFunction(action:'ajaxTechnologies', update: 'technologywrapper', onSuccess: 'updatePlatforms()', params:'\'vendorName=\' + this.value + \'&technologyName=\' + $F(\'technology\') + \'&measurementName=\' + $F(\'measurement\')' )};
-					${remoteFunction(action:'ajaxMeasurements', update: 'measurementwrapper', onSuccess: 'updatePlatforms()', params:'\'vendorName=\' + this.value + \'&technologyName=\' + $F(\'technology\') + \'&measurementName=\' + $F(\'measurement\')' )}"/>
+			onchange="${legacy.remoteFunction(action:'ajaxTechnologies', update: 'technologywrapper', onSuccess:
+					'updatePlatforms()',
```

Comment:
I don't think this `onSuccess` param is supported by the current `legacy.remoteFunction`, but we can fix that later.